course5/homework1/homework1_v2.cc

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+#include <algorithm>
+#include <cstring>
+
+int main(int argc, char** argv)
+{
+    std::sort(argv, argv + argc,
+            [](const char* s1, const char *s2){ return strcmp(s1, s2) < 0; });
```

Comment:
Use ```std::strcmp```. The unqualified version is a C-call, which is accessible only by accident.